package.json

```diff
@@ -18,14 +18,18 @@
     "extract-translations": "turbo extract-translations -- --config ../../tools/i18next-parser.config.js"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.3.1",
     "classnames": "^2.3.2",
+    "react-hook-form": "^7.46.2",
     "swr": "^2.0.1",
-    "xlsx": "^0.18.5"
+    "xlsx": "^0.18.5",
+    "zod": "^3.22.2"
   },
   "devDependencies": {
     "@babel/core": "^7.11.6",
     "@carbon/react": "~1.37.0",
     "@openmrs/esm-framework": "next",
+    "@openmrs/esm-patient-common-lib": "next",
```

Comment:
Do we need this dependency? I don't see it used in any of the code marked as coming from the chart.

Reply:
I forget why I needed it, but it was because something wasn't working... potentially something I later removed, I will take it out and see if things still work.